packages/manager/src/features/Linodes/PublicIPAddressesTooltip.tsx

```diff
@@ -8,21 +8,32 @@ const sxTooltipIcon = {
   paddingLeft: '4px',
 };
 
-export const PUBLIC_IP_ADDRESSES_TOOLTIP_TEXT =
+export const PUBLIC_IP_ADDRESSES_CONFIG_INTERFACE_TOOLTIP_TEXT =
   'The noted Public IP Addresses are provisionally reserved but not assigned to the network interfaces in this configuration profile.';
 
-export const PublicIPAddressesTooltip = (
-  <TooltipIcon
-    text={
-      <Typography>
-        {PUBLIC_IP_ADDRESSES_TOOLTIP_TEXT}{' '}
-        <Link to="https://techdocs.akamai.com/cloud-computing/docs/manage-configuration-profiles-on-a-compute-instance">
-          Learn more
-        </Link>
-        .
-      </Typography>
-    }
-    status="help"
-    sxTooltipIcon={sxTooltipIcon}
-  />
-);
+export const PUBLIC_IP_ADDRESSES_LINODE_INTERFACE_TOOLTIP_TEXT =
+  'The noted Public IP Addresses are provisionally reserved but not assigned to the network interfaces.';
```

Comment:
This copy reads a bit weird to me. Is the `the` intentional? Maybe it could be removed/replaced

```suggestion
  'The noted Public IP Addresses are provisionally reserved but not assigned to this Linode's network interfaces.';
```

```suggestion
  'The noted Public IP Addresses are provisionally reserved but not assigned to network interfaces.';
```

Reply:
can do - what are your thoughts on the copy for legacy interfaces - `'The noted Public IP Addresses are provisionally reserved but not assigned to the network interfaces in this configuration profile.';`?

maybe `"...the network interfaces in this Linode's configuration profile"` instead?

Reply:
For  legacy interfaces, I'm honestly not sure what the copy should be becausee I don't know how it works with when you have configuration profiles

Reply:
yeah, I'm guessing it's similar to Linode Interfaces where the IPs don't belong to the config's interfaces but not too sure - I'll keep the original copy since that was approved previously